public/assets/udb-logo-mobile.svg

```diff
@@ -0,0 +1,16 @@
+      />
+    </g>
+</svg>
```

Comment:
We can make React components that render SVG code out of them, that way we don't have to load them via image tags, same the component below